apps/src/sites/studio/pages/programming_expressions/show.js

```diff
@@ -6,12 +6,37 @@ import ExpandableImageDialog from '@cdo/apps/templates/lessonOverview/Expandable
 import instructionsDialog from '@cdo/apps/redux/instructionsDialog';
 import {getStore, registerReducers} from '@cdo/apps/redux';
 import {Provider} from 'react-redux';
+import {installCustomBlocks} from '@cdo/apps/block_utils';
+import {customInputTypes} from '@cdo/apps/p5lab/spritelab/blocks';
+import animationList, {
+  setInitialAnimationList
+} from '@cdo/apps/p5lab/redux/animationList';
+import {
+  valueTypeTabShapeMap,
+  exampleSprites
+} from '@cdo/apps/p5lab/spritelab/constants';
+import assetUrl from '@cdo/apps/code-studio/assetUrl';
 
 $(document).ready(() => {
   registerReducers({
-    instructionsDialog
+    instructionsDialog,
+    animationList
   });
   const store = getStore();
+  const customBlocksConfig = getScriptData('customBlocksConfig');
+  if (customBlocksConfig) {
+    store.dispatch(setInitialAnimationList(exampleSprites));
```

Comment:
nit: since this line and line 30 are specific to spritelab (but not harmful to set/invoke for all labs), could we put those lines together and add a comment? i think it would help future us spot that this is purposeful/okay even though it's only needed for spritelab and its derived labs (like poetry)